### ISSUE TITLE
Test/Framework/Runners/Console/Run.hs: Ambiguous occurrence `<>'

### ISSUE THREAD

**Issue by ghost** [batterseapower/test-framework#17]
I'm trying to compile test-framework 0.4.2.2, but am running into multiple instances of the following error (all in Run.hs):

```
Test/Framework/Runners/Console/Run.hs:48:46:
    Ambiguous occurrence `<>'
    It could refer to either `Text.PrettyPrint.ANSI.Leijen.<>',
                             imported from `Text.PrettyPrint.ANSI.Leijen' at Test/Framework/Runners/Console/Run.hs:19:1-35
                          or `Data.Monoid.<>',
                             imported from `Data.Monoid' at Test/Framework/Runners/Console/Run.hs:21:1-18
```

I'm running GHC 7.4.1 and Text.PrettyPrint.ANSI.Leijen is from ansi-wl-pprint 0.6.4.

**Comment by batterseapower** [comment]
This was fixed in test-framework 0.5 which has been out for some time. Can you use that version?

**Comment by ghost** [comment]
Thank you. That works. Unfortunately there are many packages depending on either test-framework <0.5 or monad-control <0.3 (which depends on test-framework <0.5).

I would like to have seen a 0.4.2.3, which solves just the issue with GHC 7.4.1, but I will just "backport" the patch for packaging now.

**Comment by batterseapower** [comment]
OK, I've released v0.4.2.3 as you asked.

http://hackage.haskell.org/package/test-framework-0.4.2.3

**Comment by ghost** [comment]
Thank you.
